src/main/resources/db/migration/V5__2024_04_03_1533.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE mentor_interview_type;
+
+UPDATE account SET created_by = NULL;
```

Comment:
Do we have any specific reason to keep the createdBy and the modifiedBy column types as CHAR instead of VARCHAR

Reply:
Nope. I will handle it

Reply:
actually createdBy and modifiedBy are now bigint not char.